app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -1038,6 +1037,7 @@ def rbac_user_set_record_vars(user)
     user.userid     = @edit[:new][:userid]
     user.email      = @edit[:new][:email]
     user.password   = @edit[:new][:password] if @edit[:new][:password]
+    user.update_attributes(:miq_groups => Rbac.filtered(MiqGroup.find(rbac_user_get_group_ids)))
```

Comment:
Is it ok to change to:

```diff
-user.update_attributes(:miq_groups => Rbac.filtered(MiqGroup.find(rbac_user_get_group_ids)))
+user.update_attributes(:miq_groups =>Rbac.filtered(MiqGroup.where(:id => rbac_user_get_group_ids))
```

Rbac works better with scopes than arrays. (the order of the list will potentially change, not sure if it matters here)

Reply:
:+1: @kbrock

Reply:
I will have more updates to this PR - as soon as I have my new dev environment ready